plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -2146,13 +2146,15 @@ void read_write::push_block(read_write::push_block_params&& params, next_functio
 
 void read_write::push_transaction(const read_write::push_transaction_params& params, next_function<read_write::push_transaction_results> next) {
    try {
-      auto pretty_input = std::make_shared<packed_transaction>();
+      packed_transaction_v0 input_trx_v0;
       auto resolver = make_resolver(this, abi_serializer::create_yield_function( abi_serializer_max_time ));
+      packed_transaction_ptr input_trx;
       try {
-         abi_serializer::from_variant(params, *pretty_input, std::move( resolver ), abi_serializer::create_yield_function( abi_serializer_max_time ));
+         abi_serializer::from_variant(params, input_trx_v0, std::move( resolver ), abi_serializer::create_yield_function( abi_serializer_max_time ));
+         input_trx = std::make_shared<packed_transaction>( input_trx_v0, true );
```

Comment:
Similarly, after the addition of `packed_transaction:: packed_transaction( packed_transaction_v0&&, bool legacy )` which we already need for other reasons (https://github.com/EOSIO/eos/pull/8891#discussion_r400342768), this should use `std::move(input_trx_v0)`.